### PR TITLE
fix comment on the location of scale and bias (offset) in each fused rowwise 8bit

### PR DESCRIPTION
**Pull request by jspark1105**

### CODE REVIEW

File: caffe2/operators/fused_rowwise_8bit_conversion_ops.cc [hunk 1]
```diff
@@ -47,9 +47,9 @@ OPERATOR_SCHEMA(Fused8BitRowwiseQuantizedToFloat)
     .SetDoc(R"DOC(
 De-quantizes the result of the
 FloatToFused8BitRowwiseQuantized operator. The input is expected to
-encode the scale as a 32-bit float in the first 4 bytes of each row,
-followed by the bias as a 32-bit float in the next 4 bytes, followed by
-the quantized values in the remaining bytes of the row. The output is a
+encode the scale as a 32-bit float in the second to the last 4 bytes of each
+row, followed by the bias as a 32-bit float in the next 4 bytes, and the
+quantized values in the preceding remaining bytes of the row. The output is a
```
Comment (kuttas) [inline caffe2/operators/fused_rowwise_8bit_conversion_ops.cc]: "preceding remaining" is a little confusing - do you just mean "preceding" here?

File: caffe2/operators/lengths_reducer_fused_8bit_rowwise_ops.cc [hunk 2]
```diff
@@ -28,7 +28,7 @@ OPERATOR_SCHEMA(SparseLengthsSumFused8BitRowwise)
     .SetDoc(R"DOC(
 Performs the same operation as SparseLengthsSum, but operating on
 8-bit rowwise quantized matrices with fused storage (where each row
-stores the scale, bias and then quantized values).
+stores quantized values, and then the scale and bias).
```
Comment (kuttas) [inline caffe2/operators/lengths_reducer_fused_8bit_rowwise_ops.cc]: Is it worth going into detail, e.g. "and then 8 bytes: 4 bytes for scale and 4 bytes for bias" or is that obvious given the context? (same tweak could apply for the other comment corrections below)

**Review by kuttas — APPROVED** [review]
Had 2 clarifying questions on your changes - submit if you think it's good as-is or fix nits and then submit

File: caffe2/operators/fused_rowwise_8bit_conversion_ops.cc (hunk 1 above)
Comment (jspark1105) [reply caffe2/operators/fused_rowwise_8bit_conversion_ops.cc]: Thanks for catching the typo.

File: caffe2/operators/lengths_reducer_fused_8bit_rowwise_ops.cc (hunk 2 above)
Comment (jspark1105) [reply caffe2/operators/lengths_reducer_fused_8bit_rowwise_ops.cc]: Good point. Will do.

**Pull request merged**